api/envoy/api/v3alpha/core/protocol.proto

```diff
@@ -51,6 +65,11 @@ message Http1ProtocolOptions {
   // Envoy does not otherwise support HTTP/1.0 without a Host header.
   // This is a no-op if *accept_http_10* is not true.
   string default_host_for_http_10 = 3;
+
+  // Describes how the keys for response headers should be formatted. By default, all header keys
+  // are lower cased.
+  // [#not-implemented-hide:] Hiding until both request and response formating has been implemented.
```

Comment:
So you will do docs / release notes in the follow up change?

Reply:
yep